libctru/include/3ds/services/cam.h

```diff
@@ -0,0 +1,648 @@
+    u16 awbCcmL9Right;  ///< Right camera, left color correction matrix red/green gain.
+    u16 awbCcmL9Left;   ///< Left camera, left color correction matrix red/green gain.
+    u16 awbCcmL10Right; ///< Right camera, left color correction matrix blue/green gain.
+    u16 awbCcmL10Left;  ///< Left camera, left color correction matrix blue/green gain.
+    u16 awbX0Right;     ///< Right camera, color correction matrix position threshold.
+    u16 awbX0Left;      ///< Left camera, color correction matrix position threshold.
+} CAMU_ImageQualityCalibrationData;
+
+/**
+ * @brief Stereo camera calibration data.
+ */
+typedef struct {
+    struct Flags {
+        bool isValidRotationXY; ///< Whether the X and Y rotation data is valid.
+        bool reservedFlag[3];   ///< Reserved flags for future use. (Seems unused)
+    } flags;                ///< Calibration data flags.
+    float scale;            ///< Scale to match the left camera image with the right.
+    float rotationZ;        ///< Z axis rotation to match the left camera image with the right.
+    float translationX;     ///< X axis translation to match the left camera image with the right.
+    float translationY;     ///< Y axis translation to match the left camera image with the right.
+    float rotationX;        ///< X axis rotation to match the left camera image with the right.
+    float rotationY;        ///< Y axis rotation to match the left camera image with the right.
+    float angleOfViewRight; ///< Right camera angle of view.
+    float angleOfViewLeft;  ///< Left camera angle of view.
+    float distanceToChart;  ///< Distance between cameras and measurement chart.
+    float distanceCameras;  ///< Distance between left and right cameras.
+    s16 imageWidth;         ///< Image width.
+    s16 imageHeight;        ///< Image height.
+    u8 reserved[16];        ///< Reserved data for future use. (Seems unused)
+} CAMU_StereoCameraCalibrationData;
+
+/**
+ * @brief Batch camera configuration for use without a context.
+ */
+typedef struct {
+    CAMU_CameraSelect camera : 8;           ///< Selected camera.
+    s8 exposure;                            ///< Camera exposure.
+    CAMU_WhiteBalance whiteBalance : 8;     ///< Camera white balance.
+    s8 sharpness;                           ///< Camera sharpness.
+    bool autoExposureOn;                    ///< Whether to automatically determine the proper exposure.
+    bool autoWhiteBalanceOn;                ///< Whether to automatically determine the white balance mode.
+    CAMU_FrameRate frameRate : 8;           ///< Camera frame rate.
+    CAMU_PhotoMode photoMode : 8;           ///< Camera photo mode.
+    CAMU_Contrast contrast : 8;             ///< Camera contrast.
+    CAMU_LensCorrection lensCorrection : 8; ///< Camera lens correction.
+    bool noiseFilterOn;                     ///< Whether to enable the camera's noise filter.
+    int : 8;                                ///< Padding.
```

Comment:
Well, thing is, while I like bitfields since it can ensure type safety, the problem with those is that its up to the compiler to (re)arrange them in any order. While GCC usually does what you could expect from it when using byte aligned bitfields, and that we do use those in a few other places, I'd rather use a u8 instead of bool+bitfield.
I also think you should check those structures size and check if the members are indeed in the order you expect them to be if they are to be passed to services commands.